site/tutorials/tutorial-one-objectivec.md

```diff
@@ -130,8 +117,8 @@ Lastly, we close the connection:
 
 > #### Sending doesn't work!
 >
-> If this is your first time using RabbitMQ and you don't see the "Sent"
-> message then you may be left scratching your head wondering what could
+> If this is your first time using RabbitMQ and you get logged errors at this
```

Comment:
No. There's actual logging going on, because the `RMQConnectionDelegateLogger` handles errors. Maybe "errors logged" works better?